team-fingerprints-web/src/components/BackToScreen/__tests__/BackToScreen.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import BackToScreen from "../BackToScreen";
+import { renderWithRouter } from "test/testHelpers";
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+const mockedUsedNavigate = jest.fn();
+
+jest.mock("react-router-dom", () => ({
+  ...(jest.requireActual("react-router-dom") as any),

```

Comment:
unnecessary type assertion
```suggestion
  ...(jest.requireActual("react-router-dom")),
```
also try to not use type `any`

Reply:
Right, done